packages/cfpb-forms/src/atoms/range.less

```diff
@@ -0,0 +1,56 @@
+.rangeTrackStyles() {
+  height: unit((11px / @base-font-size-px), em);
```

Comment:
For new stuff I have been using `rem` instead of `em`. Wanna replace all the `em`s here with `rem`s?